deploy/07_deploy_token_staking.ts

```diff
@@ -0,0 +1,46 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const T = await deployments.get("T")
+  const KeepTokenStaking = await deployments.get("KeepTokenStaking")
+  const NuCypherStakingEscrow = await deployments.get("NuCypherStakingEscrow")
```

Comment:
Deployment on hardhat fails with `Error: No deployment found for: NuCypherStakingEscrow` error. It looks we need to create a `package.json` dependency to the package containing the contract and create a script that will resolve it.

Reply:
c7415740d4eeff56fee0fed17ddd74e10adebbef we will deploy a stub for hardhat local network.